src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -71,6 +70,13 @@ namespace winrt::TerminalApp::implementation
         _RecalculateAndApplyTabColor();
     }
 
+    void TerminalTab::_SetToolTip(const winrt::hstring& tabTitle)
+    {
+        auto toolTip = WUX::Controls::ToolTip{};

```

Comment:
```suggestion
        WUX::Controls::ToolTip toolTip{};
```

optional change to improve code readability. we usually don't use "auto x = type{y}" because auto is only good when you _don't have to say the type_. `type x{y};` is shorter :smile: